go-controller/pkg/ovn/egressip.go

```diff
@@ -2078,19 +2078,15 @@ func (oc *Controller) checkEgressNodesReachability() {
 		}
 		oc.eIPC.allocator.Unlock()
 		for nodeName, shouldDelete := range reAddOrDelete {
-			node, err := oc.watchFactory.GetNode(nodeName)
```

Comment:
Note for reviewers: Removing this logic since its not actually adding any value, then only thing we could see is to try and pull the latest state from kubeapi, but we do have other events for nodes that get processed in case the state of kapi changes, so I don't think this GetNode is very useful here.